code/game/objects/effects/decals/crayon.dm

```diff
@@ -282,105 +305,121 @@ obj/item/scroll/attackby(obj/item/I, mob/living/carbon/human/M)
 			mightier_spell(M)
 			return
 
-//if we don't cast anything then we end up doing a normal burn.
+// If we don't cast anything then we end up doing a normal burn.
 		to_chat(M, "<span class='warning'>You ignite the scroll. It burns for a few moments before becoming ash.</span>")
 		ScrollBurn()
 		return
 
-//start of book based spells
+// Proc that controls all Book-type spells
 /obj/effect/decal/cleanable/crayon/attackby(obj/item/I, mob/living/carbon/human/M)
 	..()
 	if(istype(I, /obj/item/oddity/common/book_unholy) || istype(I, /obj/item/oddity/common/book_omega))
-		if(M.disabilities&NEARSIGHTED && is_rune && M.species?.reagent_tag != IS_SYNTHETIC)
-
+		if((M.disabilities&NEARSIGHTED || M.disabilities&BLIND) && is_rune && M.species?.reagent_tag != IS_SYNTHETIC)
```

Comment:
```suggestion
		if(M.disabilities&NEARSIGHTED && is_rune && M.species?.reagent_tag != IS_SYNTHETIC)
```
Blind is to easy

Reply:
If you guarantee to me that blind Scribes will be able to cast, then it's all good